packages/lib/modules/lbp/steps/SaleStructureStep.tsx

```diff
@@ -398,6 +404,74 @@ function UserActionsInput({ control }: { control: Control<SaleStructureForm> })
   )
 }
 
+function FeeSelection({
+  control,
+  errors,
+  feeValue,
+  setFormValue,
+}: {
+  control: Control<SaleStructureForm>
+  errors: FieldErrors<SaleStructureForm>
+  feeValue: number
+  setFormValue: UseFormSetValue<SaleStructureForm>
+}) {
+  const [value, setValue] = useState('minimum')
+
+  useEffect(() => {
+    if (feeValue !== 1.0) setValue('custom')
+  }, [feeValue])
+
+  const isBiggerThanMinimum = (fee: number) => {
+    if (fee < 1) return 'LBP swap fees must be set at or above 1.00%'
+    return true
+  }
```

Comment:
LBPs inherit from weighted pools so they also have maximum swap fee % 

I tried entering value greater than the max allowed which causes pool create tx to revert

Suggest we consider setting maximum to 10%

https://github.com/balancer/balancer-v3-monorepo/blob/1eec006f7242c3dae41dd502e286e6af7c961e6e/pkg/pool-weighted/contracts/WeightedPool.sol#L50-L51

Reply:
done